subprojects/capstone-v5.mk

```diff
@@ -4,16 +4,18 @@ WRAP_wrap_git_url:=https://github.com/capstone-engine/capstone.git
 WRAP_wrap_git_revision:=accf4df62f1fba6f92cae692985d27063552601c
 WRAP_wrap_git_patch_directory:=capstone-v5
 WRAP_wrap_git_directory:=capstone-v5
+WRAP_wrap_git_diff_files:=capstone-v5/capstone-patches/fix-x86-16.patch
 WRAP_wrap_git_depth:=1
 
 capstone-v5_all: capstone-v5
 	@echo "Nothing to do"
 
 capstone-v5:
 	git clone --no-checkout --depth=1 https://github.com/capstone-engine/capstone.git capstone-v5
-	cd capstone-v5 && git fetch --depth=1 origin accf4df62f1fba6f92cae692985d27063552601c
+	cd capstone-v5 && git fetch --depth=1 origin accf4df62f1fba6f92cae692985d27063552601c && git checkout accf4df62f1fba6f92cae692985d27063552601c
 	cd capstone-v5 && git checkout FETCH_HEAD
-	cp -f packagefiles/capstone-v5/* capstone-v5
+	cp -rf packagefiles/capstone-v5/* capstone-v5
+	for a in capstone-v5/capstone-patches/fix-x86-16.patch ; do echo "patch -d capstone-v5 -p1 < $$a" ; patch -d capstone-v5 -p1 < $$a ; done
```

Comment:
are we sure this patch is applied? because the failing test is x86-16 binary (unzip.exe)

Reply:
or maybe it wasnt applied before :?